portal-parent/pom.xml

```diff
@@ -241,13 +242,25 @@
                 <version>${hazelcast.version}</version>
             </dependency>
 
-
             <dependency>
                 <groupId>javax.validation</groupId>
                 <artifactId>validation-api</artifactId>
                 <version>${validation-api.version}</version>
             </dependency>
 
+            <!-- Remaining dependency on Markdown4l -->
+            <dependency>
+                <groupId>org.commonjava.googlecode.markdown4j</groupId>
+                <artifactId>markdown4j</artifactId>
+                <version>${markdown4j.version}</version>
+            </dependency>
+
+            <dependency>
```

Comment:
I commented out the Guava dependencies in the two POMs and it worked. 

Since it is a transitive dependency from spring integration library, I would prefer not seeing it in portal project.